spec/cohortManager.spec.js

```diff
@@ -0,0 +1,217 @@
+const CohortManager = require('../src/cohortManager.js')
+const Cohort = require('../src/cohort.js')
+const Student = require('../src/student.js')
+
+describe('The Cohort Manager should be able to', () => {
+  let cm
+
+  beforeEach(() => {
+    cm = new CohortManager()
+  })
+
+  it('create a new cohort with a name', () => {
+    // set up
+    const expected = [new Cohort('test')]
```

Comment:
I would probably add a small spec for the Cohort class; to confirm that the Cohort class constructor has default properties as you expect them to be